File: language/clojure/packages/org.clojure/clojure/core/data/operation/filter.clj
; Filter elements in which the predicate returns true

;; vector
(filter even? [0 1 2])

;; map
(defn even??
  [[k v]]
  (= 0 (rem v 2)))

(filter even?? {:k0 0, :k1 1, :k2 2})

File: language/python/modules/python/builtins/Classes/filter.py
# Returns an iterator
# %%
v = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
it = filter(lambda el: el % 2 == 0, v)
list(it)

# %%
m = {"a": 1, "b": 2, "c": 3}
it = filter(lambda kv: kv[0] != "b", m.items())
dict(it)
